content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md

```diff
@@ -84,6 +84,8 @@ kubectl diff -f https://k8s.io/examples/application/simple_deployment.yaml
 
 {{< note >}}
 `diff` uses [server-side dry-run](/docs/reference/using-api/api-concepts/#dry-run), which needs to be enabled on `kube-apiserver`.
+
+As a result, `diff` users need need write authorization to PATCH the resource.
```

Comment:
Same as above, link / content on how user can get the required permission?

Reply:
I don't think the "As a result" is super obvious for people reading this section (they're not in the same mindset as you are writing this pr ;-). But rather, maybe something along the lines of .. 
> Since dry-run requires write authorization to PATCH, `diff` users will need write authorization to use the feature

Reply:
Clarified this sentence for the reader :) and linked to the dry run authorization section.